### PR TITLE
Work up to the 10th of December

### PR DESCRIPTION
**Pull request by DanielTurner**
The following tickets have been addressed::
* C4A-202
* C4A-346
* C4A-332
* C4A-335

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,6 +1,19 @@
+<!DOCTYPE html>
 <link rel="stylesheet" href="./styles/main.scss" />
 <script src="https://unpkg.com/formiojs@latest/dist/formio.full.min.js"></script>
 
+<!-- Horrible code only impacting old browsers -->
```
Comment (duttonw) [inline src/index.html:5]: Can you update the comment for what browsers this does fix

Comment (DanielTurner) [reply src/index.html:5]: This feature detects and patches browsers that don't have fetch and the second one is those that don't have htmlTemplateElement. This includes but is not limited to Internet Explorer.

File: src/resources/health-hygiene/PDF/resources/view.html [hunk 2]
```diff
@@ -14,21 +14,21 @@ <h3>Do Food Safely</h3>
     <div class="qg-card__footer">
       <a
         class="anchor-margin"
-        href="https://dofoodsafely.health.vic.gov.au/index.php/en/"
+        href="http://dofoodsafely.health.vic.gov.au/index.php/en/"
```
Comment (duttonw) [inline src/resources/health-hygiene/PDF/resources/view.html:17]: is this website alive?

File: src/resources/home/view.html [hunk 3]
```diff
@@ -25,7 +25,7 @@ <h2 class="heading-primary">Topics to complete</h2>
 <div>
   <h3>People</h3>
   <section class="row qg-cards">
-    <article class="qg-card col-12 col-sm-6 col-lg-4">
+    <article class="qg-card qg-card__light-theme col-12 col-sm-6 col-lg-4">
       <div class="content">
         <img
           src="https://www.qld.gov.au/dev/food-pantry/dev/kyfb-home-images/Skills-and-knowledge-176606698.jpg"
```
Comment (duttonw) [inline src/resources/home/view.html:31]: TODO: images are hard coded to dev.

**Review by duttonw — APPROVED** [review]

File: src/resources/health-hygiene/PDF/resources/view.html (hunk 2 above)
Comment (DanielTurner) [reply src/resources/health-hygiene/PDF/resources/view.html:17]: We did comment wondering how they can do food safely but not website security.

Comment (duttonw) [reply src/resources/health-hygiene/PDF/resources/view.html:17]: just trying to go to it gives me
504 Gateway Time-out
The server didn't respond in time.

Comment (DanielTurner) [reply src/resources/health-hygiene/PDF/resources/view.html:17]: The server is meant to be live, it was on Thursday, and is still linked to by the vic health website.
Eg: 
https://www2.health.vic.gov.au/public-health/food-safety/food-safety-information-for-community-groups/food-safety-training-skills-knowledge/do-food-safely

Comment (duttonw) [reply src/resources/health-hygiene/PDF/resources/view.html:17]: I emailed the vic.gov that was linked on a cached copy of dofoodsafely that it did not have ssl and may have been hacked/malware/crypto miner infected as its offline.

**Pull request merged**